tests/Avalonia.Base.UnitTests/Media/MatrixTests.cs

```diff
@@ -30,6 +30,16 @@ public void Identity_Has_Inverse()
             Assert.True(matrix.HasInverse);
         }
 
+        [Fact]
+        public void Invert_Should_Work()
```

Comment:
I think the correct test should be A * A-¹ = I

Reply:
> I think the correct test should be A-¹ * A = I

I agree with this, although the way it's currently written follows the unit test concept a bit better as it doesn't depend on other functions.